docs/api/supervisor/endpoints.md

```diff
@@ -1906,6 +1906,66 @@ Shutdown the host
 
 </ApiEndpoint>
 
+<ApiEndpoint path="/host/disks/default/usage" method="get">
```

Comment:
What does `default` mean?

Reply:
Should we mention that this is the only supported path and it will return information about the data disk?